models/contacts.js

```diff
@@ -1,14 +1,14 @@
 const Contacts = (connection, Sequelize) => {
-  return connection.define('testTables', {
+  return connection.define('contacts', {
     id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
     Email: { type: Sequelize.STRING, allowNull: false },
     First_Name: { type: Sequelize.STRING, allowNull: false },
     Last_Name: { type: Sequelize.STRING, allowNull: false },
-    Phone_number: { type: Sequelize.STRING, allowNull: false },
+    Phone_number: { type: Sequelize.INTEGER, allowNull: false },
     City: { type: Sequelize.STRING, allowNull: false },
     State: { type: Sequelize.STRING, allowNull: false },
-    Last_Order_Price: { type: Sequelize.DECIMAL(13, 2), allowNull: false },
-    Last_Order_Date: { type: Sequelize.DATEONLY, allowNull: false }
+    Last_Order_Price: { type: Sequelize.INTEGER, allowNull: false },
+    Last_Order_Date: { type: Sequelize.DATE, allowNull: false }
```

Comment:
I think we would want this to be date only since the CSV only has the date

Reply:
Also I think these all need to be changed to camel case, not sure if that is going to be in another pull request